examples/kyma-mock/test.js

```diff
@@ -0,0 +1,26 @@
+var request = require('supertest');
+var exampleApp = require("./app.js")
+
+describe('tests openapi controllers', function () {
+    it('should work', function (done) {
+        exampleApp.then(function (app) {
+            describe('GET courses', function () {
+                it('should return 200', function (done) {
+                    request(app)
+                        .get('/kyma/courses')

```

Comment:
If that test works then we did something very wrong

Reply:
wrote tests for all apis